### PR TITLE
Added manifest for com.github.arshubham.cipher

### PR DESCRIPTION
**Pull request by sahilarora3117**

### CODE REVIEW

**Comment by sahilarora3117** [comment]
bot, build com.github.arshubham.cipher

**Comment by sahilarora3117** [comment]
bot, build com.github.arshubham.cipher

**Comment by sahilarora3117** [comment]
bot, build com.github.arshubham.cipher

**Comment by sahilarora3117** [comment]
bot, build com.github.arshubham.cipher

**Comment by sahilarora3117** [comment]
bot, build com.github.arshubham.cipher

File: com.github.arshubham.cipher.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "com.github.arshubham.cipher",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.36",
```
Comment (bilelmoussaoui) [inline com.github.arshubham.cipher.json:4]:
```suggestion
    "runtime-version" : "3.38",
```

File: com.github.arshubham.cipher.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
```
Comment (bilelmoussaoui) [inline com.github.arshubham.cipher.json:6]:
```suggestion
    "base-version" : "juno-20.08",
```

File: com.github.arshubham.cipher.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "command": "com.github.arshubham.cipher",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro"
```
Comment (bilelmoussaoui) [inline com.github.arshubham.cipher.json:17]:
```suggestion
```
You don't need dconf access.

**Review by bilelmoussaoui — CHANGES_REQUESTED** [review]

**Comment by sahilarora3117** [comment]
bot, build com.github.arshubham.cipher

File: com.github.arshubham.cipher.json (hunk 1 above)
Comment (sahilarora3117) [reply com.github.arshubham.cipher.json]: I'm really new to this, thanks for helping me out

File: com.github.arshubham.cipher.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "app-id": "com.github.arshubham.cipher",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.38",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno-20.08",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.arshubham.cipher",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
```
Comment (barthalion) [inline com.github.arshubham.cipher.json:11]:
```suggestion
        "--socket=fallback-x11",
```

**Comment by barthalion** [comment]
bot, build com.github.arshubham.cipher

**Pull request closed without merging**